packages/yswaps/scripts/execute-mainnet-pending-trades.ts

```diff
@@ -75,26 +76,32 @@ async function main() {
 
   let nonce = await ethers.provider.getTransactionCount(ymech.address);
 
-  console.log('[Execution] Taking snapshot of fork');
-
-  const snapshotId = (await network.provider.request({
-    method: 'evm_snapshot',
-    params: [],
-  })) as string;
-
   console.log('------------');
   for (const strategy in mainnetConfig) {
     const tradesConfig = mainnetConfig[strategy];
     console.log('[Execution] Processing trade of strategy', tradesConfig.name);
     for (const tradeConfig of tradesConfig.tradesConfigurations) {
       console.log('[Execution] Processing', tradeConfig.enabledTrades.length, 'enabled trades with solver', tradeConfig.solver);
 
-      const solver = mainnetSolversMap[tradeConfig.solver];
+      const solver = mainnetSolversMap[tradeConfig.solver] as Solver;
```

Comment:
this typing should not be necessary: `as Solver`. And in case of just being more explicit, could we type like this way?
`const solver: Solver = mainnetSolversMap[tradeConfig.solver]`

Reply:
Aktually there is no need to type this. My VSC was completely broken. Fixed.